JenkinsfileCommon.groovy

```diff
@@ -0,0 +1,16 @@
+def getAppVersion() {
```

Comment:
This file should go into "uaa-cf-release" if we're following the existing conventions.

Reply:
All Jenkinsfiles are here. And this is just common Jenkinsfiles things.